### PR TITLE
Issue 6889: Upgrading adoptOpenJDK and python3 to 3.10.5

### LINKED ISSUE #6889: Update docker base image AdoptOpenJDK to latest and python3 to 3.10.5
**Describe the bug**

- AdoptOpenJDK `11.0.11_9-alpine` was last updated an year ago. Latest version is `11.0.16.1_1-alpine`
- A security scan of Pravega image flags python3:3.9.5 with CVE-2015-20107 and CVE-22022-26488. These are fixed in python3:3.10.5

**To Reproduce**
Run a security scan like BlackDuck or similar

**Location**
https://github.com/pravega/pravega/blob/25429667a57754c9564fb879a5f734c8301d60e8/docker/pravega/Dockerfile

### PR DESCRIPTION
**Pull request by kotlasaicharanreddy**
**Change log description**  
- AdoptOpenJDK `11.0.11_9-alpine` was last updated an year ago. Latest version is `11.0.16.1_1-alpine`
- Updated python3 from `3.9.5` to `3.10.5`. This version of Python fixes CVE-2015-20107 and CVE-2022-26488

**Purpose of the change**  
Fixes #6889

**What the code does**  

- Upgrades AdoptOpenJDK to latest
- Upgrades python3 from 3.9.5 to 3.10.5

**How to verify it**  
```
root@ubuntu2004:~/pravega# docker image list
REPOSITORY               TAG                            IMAGE ID       CREATED         SIZE
pravega/pravega          0.13.0-3144.f2bc628-SNAPSHOT   8ea567e58d4f   6 minutes ago   414MB
pravega/pravega          latest                         8ea567e58d4f   6 minutes ago   414MB
pravega/pravega          0.13.0-3143.d4dac84-SNAPSHOT   1857f8e69d28   3 hours ago     412MB
adoptopenjdk/openjdk11   jre-11.0.16.1_1-alpine         f7f1c841c501   18 hours ago    149MB
root@ubuntu2004:~/pravega# docker run -it --rm --entrypoint=/bin/sh pravega/pravega:latest
/opt/pravega # apk list | grep python
WARNING: Ignoring https://dl-cdn.alpinelinux.org/alpine/v3.14/main: No such file or directory
WARNING: Ignoring https://dl-cdn.alpinelinux.org/alpine/v3.14/community: No such file or directory
python3-3.10.5-r0 x86_64 {python3} (PSF-2.0) [installed]
/opt/pravega # 
```

### CODE REVIEW

**Review by krishan-Rai — APPROVED** [review]

**Review by RaulGracia — APPROVED** [review]

File: docker/pravega/Dockerfile [hunk 1]
```diff
@@ -13,13 +13,13 @@
...
 #
-FROM adoptopenjdk/openjdk11:jre-11.0.11_9-alpine
+FROM adoptopenjdk/openjdk11:jre-11.0.16.1_1-alpine
 
 RUN apk --update --no-cache add \
     #used in readiness and liveness probes
     curl \
     #used in wait_for function
-    python3 \
+    python3=3.10.5-r0 --repository=http://dl-cdn.alpinelinux.org/alpine/v3.16/main \
```
Comment (shshashwat) [inline docker/pravega/Dockerfile:22]: this need to be changed with `3.10.7`

**Review by RaulGracia — CHANGES_REQUESTED** [review]
Blocking until fixing Python version.

**Comment by shshashwat** [comment]
@kotlasaicharanreddy , is this PR still valid? if not then we may want to close this.

**Comment by RaulGracia** [comment]
Closing, as the Temurin image does not work with glibc.

**Pull request closed without merging**